components/script/dom/file.rs

```diff
@@ -73,7 +73,8 @@ impl File {
         let ref typeString = blobPropertyBag.type_;
 
         let modified = filePropertyBag.lastModified;
-        Ok(File::new(global, BlobImpl::new_from_bytes(bytes), filename, modified, typeString))
+        let replaced_filename = DOMString::from_string(filename.replace("/", ":"));
```

Comment:
What do you mean by "spec bug"? Regarding link, this tiny replace operation is specified in the url above this function `pub fn Constructor` (which I unfortunately neglected at first)